composer.json

```diff
@@ -30,13 +30,13 @@
         "ramsey/uuid": "^4.7.3"
     },
     "require-dev": {
-        "eventsauce/test-utilities": "^3.2",
+        "eventsauce/test-utilities": "dev-main",
```

Comment:
ℹ️ Until [this PR](https://github.com/EventSaucePHP/TestUtilities/pull/2) is merged